framework/scenario/tests/contract_without_macros.rs

```diff
@@ -299,7 +299,7 @@ mod sample_adder {
     pub struct ContractBuilder;
 
     impl multiversx_sc::contract_base::CallableContractBuilder for ContractBuilder {
-        fn new_contract_obj<A: multiversx_sc::api::VMApi>(
+        fn new_contract_obj<A: multiversx_sc::api::VMApi + Send + Sync>(
```

Comment:
Ideally, the contract shouldn't know about Send/Sync. Doesn't it work without this?